<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>make_reservation_form</class>
 <widget class="QMainWindow" name="make_reservation_form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>825</width>
    <height>679</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>825</width>
    <height>630</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>825</width>
    <height>679</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Make Reservation</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="package_img">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>230</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(238, 238, 238);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>440</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border:none;
background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>825</width>
         <height>440</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QFrame" name="frame_2">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <property name="spacing">
            <number>15</number>
           </property>
           <property name="leftMargin">
            <number>20</number>
           </property>
           <item>
            <widget class="QLabel" name="label_3">
             <property name="styleSheet">
              <string notr="true">font: 700 14pt &quot;SF Pro Display&quot;;</string>
             </property>
             <property name="text">
              <string>Make Reservation</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QFormLayout" name="formLayout">
             <property name="verticalSpacing">
              <number>15</number>
             </property>
             <property name="leftMargin">
              <number>9</number>
             </property>
             <item row="0" column="0">
              <widget class="QLabel" name="label_12">
               <property name="minimumSize">
                <size>
                 <width>80</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>80</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>Customer_id</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="txt_cust_id">
               <property name="maximumSize">
                <size>
                 <width>180</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">QLineEdit{
    background-color: #D9D9D9;
	font: 10pt &quot;SF Pro Display&quot;;
    color: #404040;
    border: none;
    border-radius: 10px;
    padding: 8px 16px; 
	text-align: left;
}</string>
               </property>
               <property name="placeholderText">
                <string>C001</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_13">
               <property name="text">
                <string>Full name</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="txt_cust_full_name">
               <property name="styleSheet">
                <string notr="true">QLineEdit{
    background-color: #D9D9D9;
	font: 10pt &quot;SF Pro Display&quot;;
    color: #404040;
    border: none;
    border-radius: 10px;
    padding: 8px 16px; 
	text-align: left;
}</string>
               </property>
               <property name="placeholderText">
                <string>Enter Your Full Name</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_14">
               <property name="text">
                <string>Email</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QLineEdit" name="txt_cust_email">
               <property name="styleSheet">
                <string notr="true">QLineEdit{
    background-color: #D9D9D9;
	font: 10pt &quot;SF Pro Display&quot;;
    color: #404040;
    border: none;
    border-radius: 10px;
    padding: 8px 16px; 
	text-align: left;
}</string>
               </property>
               <property name="placeholderText">
                <string>Enter Your Email Address</string>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="label_15">
               <property name="text">
                <string>Mobile</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QLineEdit" name="txt_cust_mobile">
               <property name="styleSheet">
                <string notr="true">QLineEdit{
    background-color: #D9D9D9;
	font: 10pt &quot;SF Pro Display&quot;;
    color: #404040;
    border: none;
    border-radius: 10px;
    padding: 8px 16px; 
	text-align: left;
}</string>
               </property>
               <property name="placeholderText">
                <string>Enter Your Mobile No.</string>
               </property>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="QLabel" name="label_16">
               <property name="text">
                <string>No of Pax</string>
               </property>
              </widget>
             </item>
             <item row="4" column="1">
              <widget class="QSpinBox" name="cust_no_of_pax">
               <property name="maximumSize">
                <size>
                 <width>100</width>
                 <height>30</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">font: 12pt &quot;SF Pro Display&quot;;
background-color: rgb(217, 217, 217);
color: rgb(0, 0, 0);
border-radius: 10px;
    padding: 8px 16px; </string>
               </property>
              </widget>
             </item>
             <item row="5" column="1">
              <widget class="QLabel" name="cust_tot_price">
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>100</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">font: 700 12pt &quot;SF Pro Display&quot;;</string>
               </property>
               <property name="text">
                <string>-- Toal Price --</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>2</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="label_17">
             <property name="text">
              <string>Reservation date</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="txt_cust_date">
             <property name="maximumSize">
              <size>
               <width>180</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QLineEdit{
    background-color: #D9D9D9;
	font: 10pt &quot;SF Pro Display&quot;;
    color: #404040;
    border: none;
    border-radius: 10px;
    padding: 8px 16px; 
	text-align: left;
}</string>
             </property>
             <property name="placeholderText">
              <string>Select from Calandar</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="frame_3">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>350</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <property name="rightMargin">
            <number>20</number>
           </property>
           <item>
            <widget class="QLabel" name="label">
             <property name="styleSheet">
              <string notr="true">font: 700 12pt &quot;SF Pro Display&quot;;</string>
             </property>
             <property name="text">
              <string>Reservation date</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCalendarWidget" name="calendarWidget">
             <property name="styleSheet">
              <string notr="true">QCalendarWidget QToolButton {
    background-color: #ffffff;
    color: #000000;
    border: none;
    padding: 5px;
    min-width: 0;
}</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Minimum</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>108</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QFrame" name="frame_19">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_9">
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <spacer name="horizontalSpacer_7">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>23</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="btn_clear_2">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>120</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">/* Default button style */
QPushButton {
    background-color: #C79C99;
	font: 10pt &quot;SF Pro Display&quot;;
    color: #00000;
    border: none;
    border-radius: 5px;
    padding: 8px 16px; 

}

/* Button style on hover */
QPushButton:hover {
 background-color: #46464b;
	color: #ffffff;
	font-weight: bold;
}

/* Button style when pressed */
QPushButton:pressed {
    background-color: #AE544F;
 	color: #FFFFFF;
}

QPushButton:checked {
	background-color: #AE544F;
 	color: #FFFFFF;
}</string>
                </property>
                <property name="text">
                 <string>Clear</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_8">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>2</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="btn_save_reservation">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>120</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">/* Default button style */
QPushButton {
    background-color: #3474D4;
	font: 10pt &quot;SF Pro Display&quot;;
    color: #ffffff;
    border: none;
    border-radius: 5px;
    padding: 8px 16px; 
text-align: center;


}

/* Button style on hover */
QPushButton:hover {
 background-color: #A2DCD7;
	color: #ffffff;
	font-weight: bold;
}

/* Button style when pressed */
QPushButton:pressed {
    background-color: #17396D;
 	color: #FFFFFF;
}

QPushButton:checked {
	background-color: #17396D;
 	color: #FFFFFF;
}</string>
                </property>
                <property name="text">
                 <string>Save</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_9">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>1</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btn_clear_2</sender>
   <signal>clicked()</signal>
   <receiver>txt_cust_full_name</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>551</x>
     <y>632</y>
    </hint>
    <hint type="destinationlabel">
     <x>263</x>
     <y>344</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_clear_2</sender>
   <signal>clicked()</signal>
   <receiver>txt_cust_email</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>603</x>
     <y>628</y>
    </hint>
    <hint type="destinationlabel">
     <x>347</x>
     <y>407</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_clear_2</sender>
   <signal>clicked()</signal>
   <receiver>txt_cust_mobile</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>593</x>
     <y>632</y>
    </hint>
    <hint type="destinationlabel">
     <x>398</x>
     <y>443</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_clear_2</sender>
   <signal>clicked()</signal>
   <receiver>cust_no_of_pax</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>620</x>
     <y>633</y>
    </hint>
    <hint type="destinationlabel">
     <x>207</x>
     <y>501</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_clear_2</sender>
   <signal>clicked()</signal>
   <receiver>txt_cust_date</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>568</x>
     <y>641</y>
    </hint>
    <hint type="destinationlabel">
     <x>181</x>
     <y>629</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
